retis/src/process/cli/pcap.rs

```diff
@@ -217,10 +303,6 @@ pub(crate) struct Pcap {
     pub(super) out: Option<PathBuf>,
     #[arg(default_value = "retis.data", help = "File from which to read events")]
     pub(super) input: PathBuf,
-}
-#[derive(Args, Debug, Default)]
-#[group(required = true, multiple = false)]
-pub(crate) struct PcapCmd {
```

Comment:
Removing this makes commands like `retis pcap --list --probe foo` not to fail anymore. Previously an help message was shown. This is handy as the `pcap` sub-command has different incompatible modes.

Reply:
:+1:

This could be in `retis print`.